scripts/test/updateAtlas.js

```diff
@@ -22,7 +22,7 @@ async function main() {
         throw new Error("Cannot find a tests/testProject. Did you run the script in the widget folder?");
     }
 
-    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource/atlas*");
+    rm("-rf", "tests/testProject/theme", "tests/testProject/themesource/atlas_ui_resources");
```

Comment:
I don't recall any specific reasoning behind `atlas*`. To double check, I've asked Isa.

Reply:
I think it has to do with custom modules with a name starting with "atlas" but I don't know what use case this has

Reply:
Unless there's a good reason for it I'd prefer specificity (atlas_ui_resources)

Reply:
See this [convo](https://mendix.slack.com/archives/GV93BNVDM/p1614171528008500?thread_ts=1614160827.001700&cid=GV93BNVDM)